Fails to compile on Windows due to missing SystemConfiguration library

* [x] I've read, understood, and done my best to follow the [*CONTRIBUTING guidelines](https://github.com/Alamofire/Alamofire/blob/master/CONTRIBUTING.md).

## What did you do?

Attempted to compile Alamofire on Windows

## What did you expect to happen?

The library compiles properly

## What happened instead?

```
D:\a\redacted\redacted\desktop_client\shared\redacted\.build\checkouts\Alamofire\Source\NetworkReachabilityManager.swift:28:8: error: no such module ‘SystemConfiguration'

import SystemConfiguration
            ^
```
## Alamofire Environment

**Alamofire Version:**
5.4.3
**Dependency Manager:**
Swift Package Manager
**Xcode Version:**
N/A
**Swift Version:**
5.3.0
**Platform(s) Running Alamofire:**
Microsoft Windows Server 2019 10.0.17763
**macOS Version Running Xcode:**
N/A
## Demo Project

Any example project using SPM will work fine

--- a/Source/NetworkReachabilityManager.swift
+++ b/Source/NetworkReachabilityManager.swift
@@ -22,7 +22,7 @@
 //  THE SOFTWARE.
 //
 
-#if !(os(watchOS) || os(Linux))
+#if !(os(watchOS) || os(Linux) || os(Windows))
 
 import Foundation
 import SystemConfiguration

